test/expect-tests/memo/main.ml

```diff
@@ -2164,5 +2164,5 @@ let%expect_test "implicit output bug. storage cell is reused between runs" =
   set_invalidate_print_run ();
   [%expect {| Some [ "x" ] |}];
   set_invalidate_print_run ();
-  [%expect {| Some [ "x"; "x" ] |}]
+  [%expect {| Some [ "x" ] |}]
```

Comment:
Let's also edit the name of the test now that it's now a bug. Internally, I wrote this:

```
let%expect_test "Ensure that implicit output storage cell is not reused between runs" =
```